iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TerminalAdapter.java

```diff
@@ -16,81 +16,96 @@
  */
 public class TerminalAdapter extends AbstractAdapter<Terminal> implements Terminal {
 
+    private TerminalBusBreakerViewAdapter busBreakerView;
+    private TerminalBusViewAdapter busView;
+    private TerminalNodeBreakerViewAdapter nodeBreakerView;
```

Comment:
Same question than in the previous PR: should we keep the views as neested class to fit the IIDM API?

Reply:
I will make theses fields as nested classes